content/blog/2024-05-23-linebender-roadmap.md

```diff
@@ -0,0 +1,191 @@
++++
+title = "Roadmap for the next three months of work on Linebender"
+authors = ["Olivier Faure"]
+date = "2024-05-23 18:00:00"
++++
+
+On May 7 and 8, 2024, folks from the Linebender team went to [the RustNL conference](https://2024.rustnl.org/) for two days, followed by two days of Unconference.
+
+By that point, we all felt fairly exhausted, but Raph Levien being the cruel taskmaster that he is, he still made us work one extra day, and so we came together in our hotel's conference room to discuss the Linebender project, and what we wanted from it.
+
+We'd initially thought of that this day would be an informal day of unstructured chatting about the ecosystem, but it fairly quickly turned into a semi-formal planning exercise, where we tried to figure out a roadmap for the next few months.
+
+What follows is a comprehensive summary of what exactly we're planning to do. We've tried to focus on blockers, where a missing feature in on project is stalling progress on other project, because those are the hardest barriers to coordination, and we wanted to take advantage of being in the same room together to knock them down.
+
+- **Priority:** 
```

Comment:
I'm not quite sure I follow what this section is doing here. Is the intention to fill out a brief description of these fields here?

Reply:
Oh yeah, that's a draft thing.